src/OrbitGl/CaptureWindow.cpp

```diff
@@ -505,6 +512,15 @@ void CaptureWindow::DrawScreenSpace() {
   }
 }
 
+void CaptureWindow::UpdateRightMargin(float margin) {
+  {
+    if (right_margin_ != margin) {
+      right_margin_ = margin;
+      time_graph_->RequestUpdate();
+    }
+  }
+}
```

Comment:
Nit: There is one set of braces too much

Reply:
haha, thanks. I'm erasing the function by the way